src/core/qwindowkit_windows.h

```diff
@@ -125,15 +125,33 @@ namespace QWK {
         QString stringValue(QStringView subKey) const;
         QPair<DWORD, bool> dwordValue(QStringView subKey) const;
 
+        template<typename T>
+        std::optional<T> value(QStringView subKey) const { return {}; }
```

Comment:
I see you don't want this function to be called directly, so we can use static_cast(false, "xxx") to ensure this.